python_module/task1/calc.py

```diff
@@ -0,0 +1,32 @@
+"""This is a module for mathematical operations"""
+
```

Comment:
PEP 8: E302 expected 2 blank lines, found 1
Please check amount of blank lines before class declaration

Reply:
Fixed